modules/file-rotation/file-rotation.c

```diff
@@ -24,11 +24,34 @@
 #include "file-rotation.h"
 #include "modules/affile/file-signals.h"
 #include "driver.h"
+#include "time.h"
 
 static void
 _slot_file_rotation(FileRotationPlugin *self, FileFlushSignalData *data)
 {
   msg_trace("File rotation signal received", evt_tag_str("filename", data->filename));
+
+  if(data != NULL)
```

Comment:
NULL should never happen here.
If we want to make sure that the API is used correctly, we can add `g_assert(data);` instead of this condition.

Reply:
OK yes have missed that point.